### ISSUE TITLE
Install Ubuntu 16.04 either PPA or gdebi doesn't work

### ISSUE THREAD

**Issue by birchcode** [qdore/Mutate#136]
Tried installing both ways, here is via the gdebi method:

```
% sudo gdebi Mutate-2.3.deb 
Reading package lists... Done
Building dependency tree        
Reading state information... Done
Reading state information... Done
This package is uninstallable
Dependency is not satisfiable: libboost-regex1.55-dev
```

**Comment by mgolisch** [comment]
yeah your version doesnt have boost 1.55, i made some packages some time ago.
Ill dig up my build vm and reupload the debs.

**Comment by mgolisch** [comment]
you can try http://buildout.apps.mgolisch.eu/xenial_result/mutate_2.4~precise~NoolsLab.com_amd64.deb

should work on xenial

**Comment by ckruse** [comment]
Works on xenial. Thanks.

**Comment by ranjitc** [comment]
hey, anything for x86? i get a wrong architecture error using gdebi

**Comment by mgolisch** [comment]
no i never build any packages for x86, do people still have non amd64 x86 cpus?

**Comment by wzpan** [comment]
@mgolisch Good job! :+1:

**Comment by ranjitc** [comment]
@mgolisch hi, i have a few old computers that i'm converting to linux. was looking for an alfred replacement as i totally love that on osx. i'm a linux noob and have had some trouble compiling packages in the past, so was wary of compiling it myself. i'll give it a try.

**Comment by kleinfreund** [comment]
> you can try http://buildout.apps.mgolisch.eu/xenial_result/mutate_2.4~precise~NoolsLab.com_amd64.deb
> 
> should work on xenial

@mgolisch I am able to install this on Ubuntu 16.04 however opening preferences results in a `Cannot open ini file` error without any further information.

**Comment by samundra** [comment]
Do you have this file `https://github.com/qdore/Mutate/blob/master/config/config.ini` in Mutate insstalled directory? I am currently on mac so can't give you exact file location. But this is the configuration that mutate is looking for. Try to create one manually if its not already there.

**Comment by kleinfreund** [comment]
Within `~/.config/Mutate`, there is only the file `frequence`. No other files.

What I needed to do in addition:

```
git clone https://github.com/qdore/Mutate.git
cd Mutate
cp -R config/* ~/.config/Mutate
chmod -R a+x ~/.config/Mutate/scripts
chmod -R a+w ~/.config/Mutate
sed -i "s|{home}|$HOME|g" ~/.config/Mutate/config.ini
```

**Comment by samundra** [comment]
try `find . / -name 'Mutate'` and check if you have it there.

**Comment by samundra** [comment]
@kleinfreund Cheers, you got it working I guess.
